stock/forms.py

```diff
@@ -9,8 +9,8 @@ class Meta:
         widgets = {
             'name': forms.TextInput(attrs={'placeholder': 'Nombre del producto', 'maxlength': '200', 'required': 'true'}),
             'model': forms.TextInput(attrs={'placeholder': 'Modelo del producto', 'maxlength': '200'}),
-            'quantity': forms.NumberInput(attrs={'placeholder': 'Cantidad', 'step': '1', 'required': 'true'}),
-            'amount': forms.NumberInput(attrs={'placeholder': 'Precio', 'step': "0.01"}),
+            'quantity': forms.NumberInput(attrs={'placeholder': 'Cantidad', 'step': '1', 'required': 'true', 'min': '0'}),
+            'amount': forms.NumberInput(attrs={'placeholder': 'Precio', 'step': "0.1", 'min': '0'}),
```

Comment:
I would leave here the step as it was (0.01), because of two decimal prices, like for example, $9.95. With the step as it is now, you can either put an amount of $9.90 or $10, but not $9.95.